src/compiler/types.ts

```diff
@@ -662,6 +662,7 @@ namespace ts {
         initializer?: Expression;           // Optional initializer
     }
 
+    // @kind(SyntaxKind.PropertyDeclaration)

```

Comment:
We don't really need these comments, since `kind` is now part of the declaration (on line 667).